packages/react/src/component-tests/IcCard/IcCard.cy.tsx

```diff
@@ -111,15 +119,17 @@ describe("IcCard", () => {
         <IcCard heading="Card" message="This is a static card"></IcCard>
       </a>
     );
+
     cy.checkHydrated("ic-card");
     cy.findShadowEl("ic-card", TYPOGRAPHY_SELECTOR)
       .contains("This is a static card")
       .should(BE_VISIBLE);
+
+    cy.checkA11yWithWait();
     cy.compareSnapshot({
-      name: "cardTextVerify",
-      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+      name: "text-verify",
```

Comment:
Again, I think this isn't named very well. I think it should be to do with what the screenshot is, which seems to be a card wrapped in an `<a>`